solution,status,score,%test passed,time,memory,subtask_0,subtask_1,subtask_2,subtask_3,subtask_4
cern_deepseek-reasoner_0.cpp,RUNTIME_ERROR,0,0.00%,1.570,12.85546875,0,0,0,0,0
cern_deepseek-reasoner_1.cpp,RUNTIME_ERROR,0,0.00%,1.570,12.95703125,0,0,0,0,0
cern_deepseek-reasoner_2.cpp,RUNTIME_ERROR,0,0.00%,1.490,12.921875,0,0,0,0,0
cern_deepseek-reasoner_3.cpp,TIME_LIMIT_EXCEEDED,0,0.00%,4.430,41.13671875,0,0,0,0,0
cern_deepseek-reasoner_4.cpp,TIME_LIMIT_EXCEEDED,0,0.00%,4.410,161.23046875,0,0,0,0,0
cern_deepseek-reasoner_5.cpp,TIME_LIMIT_EXCEEDED,0,10.71%,4.410,41.08984375,0,0,0,0,0
cern_deepseek-reasoner_6.cpp,RUNTIME_ERROR,0,25.00%,1.430,14.765625,0,0,0,0,0
cern_deepseek-reasoner_7.cpp,TIME_LIMIT_EXCEEDED,0,0.00%,4.470,17.13671875,0,0,0,0,0
